tests/db_level_on_delete/models.py

```diff
@@ -0,0 +1,78 @@
+from django.db import models
+
+
+class Foo(models.Model):
+    """Initial model named Foo"""
+
+    pass
```

Comment:
`pass` is technically not required when the class has a docstring. I think it would be tidier to remove it.